rel/overlay/etc/default.ini

```diff
@@ -699,3 +699,7 @@ compaction = false
 ; log every generated trace by including the following:
 ;
 ; all = (#{}) -> true
+
+[couch_rate.views]
```

Comment:
This would be complex to implement. One idea that I have is to search for the right section. The API might looks like the following:
```
    Limiter = couch_rate:create_if_missing({DbName, DDocId}, [
        "views", Mrst#mrst.language]),
```

The implementation would try different sections in the following order:
1. `couch_rate.views.query` (or `couch_rate.views.javascript`)
2. `couch_rate.views`
3. `couch_rate`

The section once found would be used in its entirety i.e. we wouldn't be inheriting individual keys.